tests/components/figure.test.ts

```diff
@@ -38,6 +39,21 @@ function createFigure(
   });
 }
 
+const anchorSelectors = {
+  top: ".o-anchor.o-top",
```

Comment:
top, right, bottom and left are not necessary. If we add them, we will have to maintain it for no reason.

Reply:
This comment is marked as resolved but without changes nor explanation. Is it intended ?